backend/src/Designer/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -231,9 +234,13 @@ public List<string> GetLanguages()
             foreach (string directoryFile in directoryFiles)
             {
                 string fileName = Path.GetFileName(directoryFile);
-                string[] nameParts = fileName.Split('.');
-                languages.Add(nameParts[1]);
-                languages.Sort(StringComparer.Ordinal);
+                if (Regex.IsMatch(fileName, TextResourceFileNameRegEx))
```

Comment:
Regex is not needed here. Directory.GetFiles which used accepts search pattern as argument.